### PR TITLE
[Quanta/platform] Update IX1B to support Stretch

### PR DESCRIPTION
**Pull request by jonathantsai-qci**
What I did
Update IX1B's platform folders to support Stretch
How I did it
Follow https://github.com/Azure/SONiC/wiki/Porting-Guide
How to verify it
Follow https://github.com/Azure/SONiC/wiki/Command-Reference
to verify hardware information can get from CLI correctly

Signed-off-by: Jonathan Tsai <jonathan.tsai@quantatw.com>

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-quanta/ix1b-32x/modules/pmbus.h [hunk 1]
```diff
@@ -22,109 +22,113 @@
 #ifndef PMBUS_H
 #define PMBUS_H
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-quanta/ix1b-32x/modules/pmbus.h]: is this from linux kernel source tree? if yes, then why not use it directly?

Comment (cpchangtw) [reply platform/broadcom/sonic-platform-modules-quanta/ix1b-32x/modules/pmbus.h]: Yes it is from kernel source, as u can see other vendors also use this way to include this header file, Is there any other way to achieve this goal?

**Comment by jonathantsai-qci** [comment]
Hi Sirs,
Please help review and merge it. Thanks a lot.

File: platform/broadcom/sonic-platform-modules-quanta/ix1b-32x/modules/pmbus.h (hunk 1 above)
Comment (jonathantsai-qci) [reply platform/broadcom/sonic-platform-modules-quanta/ix1b-32x/modules/pmbus.h]: @lguohan
I modified pmbus driver to use it directly. Please help review and merge codes

**Comment by jonathantsai-qci** [comment]
Hi Sirs,
It's for a long time to pull this request. Because this Fork is more different than now master, I can close this and fork new master to pull new request for your review.
Please advise and thanks a lot.

**Pull request merged**